app/schemas/data.py

```diff
@@ -9,48 +9,101 @@ class CreateData(BaseModel):
 
     date: datetime.date
     time: datetime.time
-    nuts3: str
-    nuts2: str
-    temperature_air: Optional[float]
-    relative_humidity: Optional[float]
+
+    parcel_location: Optional[str]
+
+    atmospheric_temperature: Optional[float]
+    atmospheric_temperature_daily_min: Optional[float]
+    atmospheric_temperature_daily_max: Optional[float]
+    atmospheric_temperature_daily_average: Optional[float]
+    atmospheric_relative_humidity: Optional[float]
+    atmospheric_pressure: Optional[float]
+
     precipitation: Optional[float]
-    wind_speed: Optional[float]
-    wind_direction: Optional[float]
+
+    average_wind_speed: Optional[float]
+    wind_direction: Optional[str]
     wind_gust: Optional[float]
-    atmospheric_pressure: Optional[float]
-    relative_humidity_canopy: Optional[float]
-    temperature_canopy: Optional[float]
+
+    leaf_relative_humidity: Optional[float]
+    leaf_temperature: Optional[float]
+    leaf_wetness: Optional[float]
+
+    soil_temperature_10cm: Optional[float]
+    soil_temperature_20cm: Optional[float]
+    soil_temperature_30cm: Optional[float]
+    soil_temperature_40cm: Optional[float]
+    soil_temperature_50cm: Optional[float]
+    soil_temperature_60cm: Optional[float]
+
     solar_irradiance_copernicus: Optional[float]
 
+    dataset_id: int
+
 class UpdateData(BaseModel):
     model_config = ConfigDict(from_attributes=True)
 
-    nuts3: Optional[str]
-    nuts2: Optional[str]
-    temperature_air: Optional[float]
-    relative_humidity: Optional[float]
+    parcel_location: Optional[str]
+
+    atmospheric_temperature: Optional[float]
+    atmospheric_temperature_daily_min: Optional[float]
+    atmospheric_temperature_daily_max: Optional[float]
+    atmospheric_temperature_daily_average: Optional[float]
+    atmospheric_relative_humidity: Optional[float]
+    atmospheric_pressure: Optional[float]
+
     precipitation: Optional[float]
-    wind_speed: Optional[float]
-    wind_direction: Optional[float]
+
+    average_wind_speed: Optional[float]
+    wind_direction: Optional[str]
     wind_gust: Optional[float]
-    atmospheric_pressure: Optional[float]
-    relative_humidity_canopy: Optional[float]
-    temperature_canopy: Optional[float]
+
+    leaf_relative_humidity: Optional[float]
+    leaf_temperature: Optional[float]
+    leaf_wetness: Optional[float]
+
+    soil_temperature_10cm: Optional[float]
+    soil_temperature_20cm: Optional[float]
+    soil_temperature_30cm: Optional[float]
+    soil_temperature_40cm: Optional[float]
+    soil_temperature_50cm: Optional[float]
+    soil_temperature_60cm: Optional[float]
+
     solar_irradiance_copernicus: Optional[float]
 
 
 class DataDB(BaseModel):
```

Comment:
Is CreateData and DataDB are similar or the same maybe makes sense to create a hierarchy structure to avoid copy  paste the same values

Reply:
DataDB is actually unused, judging by both PRs, so I can remove it entirely for now, and add it again later if it is needed